src/instance.js

```diff
@@ -553,110 +518,113 @@ Instance.prototype.cluster = function(name) {
  * });
  *
  * //-
- * // To control how many API requests are made and page through the results
- * // manually, set `autoPaginate` to false.
- * //-
- * const callback = function(err, clusters, nextQuery, apiResponse) {
- *   if (nextQuery) {
- *     // More results exist.
- *     instance.getClusters(nextQuery, calback);
- *   }
- * };
- *
- * instance.getClusters({
- *   autoPaginate: false
- * }, callback);
- *
- * //-
  * // If the callback is omitted, we'll return a Promise.
  * //-
  * instance.getClusters().then(function(data) {
  *   const clusters = data[0];
  * });
  */
-Instance.prototype.getClusters = function(query, callback) {
+Instance.prototype.getClusters = function(gaxOptions, callback) {
   var self = this;
 
-  if (is.function(query)) {
-    callback = query;
-    query = {};
+  if (is.function(gaxOptions)) {
+    callback = gaxOptions;
+    gaxOptions = {};
   }
 
-  var protoOpts = {
-    service: 'BigtableInstanceAdmin',
-    method: 'listClusters',
-  };
-
-  var reqOpts = extend({}, query, {
+  var reqOpts = {
     parent: this.id,
-  });
-
-  this.request(protoOpts, reqOpts, function(err, resp) {
-    if (err) {
-      callback(err, null, null, resp);
-      return;
-    }
-
-    var clusters = resp.clusters.map(function(clusterObj) {
-      var cluster = self.cluster(clusterObj.name);
-      cluster.metadata = clusterObj;
-      return cluster;
-    });
+  };
 
-    var nextQuery = null;
+  this.bigtable.request(
+    {
+      client: 'BigtableInstanceAdminClient',
+      method: 'listClusters',
+      reqOpts: reqOpts,
+      gaxOpts: gaxOptions,
+    },
+    function(err, resp) {
+      if (err) {
+        callback(err);
```

Comment:
What determines when we pass additional data with the error?

Reply:
The GAPIC layer only returns `err`, so we only give it back as such. If there are places that say they return more, it's probably a mistake.